packages/e2e-tests/playwright.config.ts

```diff
@@ -36,6 +22,7 @@ const config: PlaywrightTestConfig = {
   use: {
     launchOptions: {
       slowMo: SLOW_MO ? parseInt(SLOW_MO, 10) : 0,
+      executablePath: "/home/holger/.replay/runtimes/chrome-linux/chrome",
```

Comment:
The changes in this file seem unrelated to the rest of the PR, but this one in particular seems accidental :)